new/PRS12948MalakaMe.xml

```diff
@@ -0,0 +1,67 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS12948MalakaMe" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Malʿaka Mǝḥrat Yāred Gǝrmāye</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2020-04-06">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">መልዓከ፡ ምሕረት፡ ያሬድ፡ ግርማዬ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Malʿaka Mǝḥrat Yāred Gǝrmāye</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic">deacon</occupation>
+                    <occupation type="ecclesiastic">Administrator of Č̣alaqot Śǝllāse</occupation>
+                    <floruit when="1975"/>
+                    <residence>
+                        <placeName ref="LOC2091Calaqo">Č̣alaqot Śǝllāse</placeName>
+                    </residence>
+                </person>
+                <!--- <listRelation>
+                  Malʿaka Mǝḥrat Yāred Gǝrmāye owned the MS from which ms BDLaethd18 was copied: 
+                        this is not mentioned in the MS (so it's not a proper attestation), but in the descriptive paper attached to BDLaethd18. 
+                        What relation better fits this circumstance?      <relation name="" active="PRS12948MalakaMe" passive=""/>
```

Comment:
I'm not sure a relation would be the best way to state this...there is the very broad dcterms:relation, which should only be used as a last recourse. Maybe this is something that would best be explained paragraphically in the manuscript description or here in `<person>` (probably in `<floruit>` or `<occupation>` then)